### ISSUE TITLE
issues about code

### ISSUE THREAD

**Issue by 1601214542** [yunjey/StarGAN#16]
hello, I saw your code of papar "StarGAN". I have some issues.

1 In data_loder.py line 43 (random.shuffle(lines)), each time you get_loader, the lines will be newly shuffled, so these train and test samples may be duplicate ?

2 in paper, equation (6), I saw the first item of loss G is L_{adv}, but in the code, I only saw loss of D(G(x,c)). I have understood that the first item of the equation 6  has no need to put into the loss function, but how about the last item. 

Thank you.

**Comment by yunjey** [comment]
@1601214542 Thank you for your question.

For 1, that's my mistake. I added the code "`random.seed(1234)`" to data_loader.py now.

For 2, you can find the generator's adversarial loss [here](https://github.com/yunjey/StarGAN/blob/master/solver.py#L335). It's a WGAN Loss which is a variation of the original GAN. I mentioned this in Section 4 of the paper.
